pay-api/src/pay_api/services/invoice.py

```diff
@@ -318,3 +334,13 @@ def get_invoices(payment_identifier: str, skip_auth_check: bool = False):
     def _check_for_auth(dao):
         # Check if user is authorized to perform this action
         check_auth(dao.account.corp_number, one_of_roles=ALL_ALLOWED_ROLES)
+
+    @staticmethod
+    def calculate_transaction_fees(payment_system_code: str, corp_type_code: str):
```

Comment:
just wondering if this BCOL scenaio can be covered in a unit test ?

Reply:
I will add test cases for transaction fees when we have the BCOL integration as it would makes sense only then.